Added third type of JS mixin (#5630)
magento/devdocs: guides/v2.2/javascript-dev-guide/javascript/js_mixins.md

});
```

#### Extend JS Object

> The last kind of JS mixin is when the base Javscript file returns an object. In this case, a wrapper is necessary. The following is an example of a mixin that extends the `setHash` function of [step navigator object][]. Here, `_super()` is the base method which can be called if needed.

Reviewer: To make this statement more future-proof, let's adjust the description slightly:

```suggestion
Another use-case of JS mixin is when the base Javascript file returns an object. In this case, a wrapper is necessary. The following is an example of a mixin that extends the `setHash` function of [step navigator object][]. Here, `_super()` is the base method that can be called if needed.
```

Thank you!

> Another use-case of JS mixin is when the base Javascript file returns an object. In this case, a wrapper is necessary. The following is an example of a mixin that extends the `setHash` function of [step navigator object][]. Here, `_super()` is the base method that can be called if needed.

Reviewer 2: ```suggestion
Another use-case for the JS mixin is when the base Javascript file returns an object. In this case, a wrapper is necessary. The following example mixin extends the `setHash` function of [step navigator object][]. Here, `_super()` is the base method that can be called if needed.
```

### Example

> The following is an example of a `requirejs-config.js` file that adds the `columns-mixin`, `modal-widget-mixin`, and `step-navigator-mixin` mixins, defined in the previous examples, to the [grid column component][], [modal widget][], and [step navigator object][].

Reviewer 2: ```suggestion
The following is an example of a `requirejs-config.js` file that adds the `columns-mixin`, `modal-widget-mixin`, and `step-navigator-mixin` mixins, which were defined in the previous examples, to the [grid column component][], [modal widget][], and [step navigator object][].
```

};
```

The following is an example of a `requirejs-config.js` file that adds the `step-navigator-mixin` mixin defined in the previous example, to the [step navigator object][].

**File:** `OrangeCompany/Sample/view/frontend/requirejs-config.js`

```javascript

> var config = {

Reviewer: Please, merge this example with the previous one. That's fair that it's applicable only for the frontend area. However, multiplying similar examples might be an overhead for the reader. So, in this particular case, we can move this declaration to the base area (just add the declaration to the previous example)

Author: Fixed it, thank you!
